tests/libtest/test_rhv.py

```diff
@@ -69,3 +68,19 @@ def test_p_stop_and_start():
     # rhv_plfrm.start_nodes(node)
     # status = rhv_plfrm.rhv.get_vm_status(vm_obj)
     # assert "up" == str(status), f"Status of {vm_name} is {status}"
+
+@libtest
+@rhv_platform_required
+def test_restart_by_stop_and_start():
+    """
+    Test of RHV restart_nodes_by_stop_and_start method implementation
+    """
+    rhv_plfrm = RHVNodes()
+    nodes = get_node_objs()
+    logger.info(f"nodes are: {nodes}")
+    node = [nodes[4]]
```

Comment:
Why did you pick `nodes[4]`? I think it will be better to choose a random node(for example:
`node = random.choice(nodes)`).

Reply:
Now, this code is not part of this PR. Marking it as resolved.